esrally/mechanic/telemetry.py

```diff
@@ -31,7 +31,8 @@
 
 def list_telemetry():
     console.println("Available telemetry devices:\n")
-    devices = [[device.command, device.human_name, device.help] for device in [JitCompiler, Gc, FlightRecorder, PerfStat, NodeStats, RecoveryStats]]
+    devices = [[device.command, device.human_name, device.help] for device in [JitCompiler, Gc, FlightRecorder, PerfStat,
```

Comment:
Should we also update the [docs](https://esrally.readthedocs.io/en/stable/telemetry.html) where we show an example of the listed telemetry devices?